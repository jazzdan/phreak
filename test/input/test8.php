<?php

class A {
    static $x = 15;
    static $y =  2;
    static $z =  3;
    public function __construct() {
        $this->e = "eee";
        $this->f = "fff";
        $this->g = "\n";
    }
    public function f2() {
        $this->e = "eee";
        $this->f = "fff";
        $this->g = "\n";
    }
    public function f3() {
        $this->e = "eee";
        $this->f = "fff";
        $this->g = "\n";
    }
    static $x2 = 150;
    static $y2 =  20;
    static $z2 =  30;
}
