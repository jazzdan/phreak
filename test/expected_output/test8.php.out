class A
{
    public static $x = 15
    public static $y = 2
    public static $z = 3
    public function __construct()
    {
        $this->e = "eee";
        $this->f = "fff";
        $this->g = "\n";
    }

    public function f2()
    {
        $this->e = "eee";
        $this->f = "fff";
        $this->g = "\n";
    }

    public function f3()
    {
        $this->e = "eee";
        $this->f = "fff";
        $this->g = "\n";
    }

    public static $x2 = 150
    public static $y2 = 20
    public static $z2 = 30
}



